dockers/release/Dockerfile

```diff
@@ -38,11 +36,14 @@ RUN \
         unzip ${LIGHTNING_VERSION}.zip ; \
         mv lightning-*/ lightning ; \
         rm *.zip ; \
-    fi && \
+    fi
     # otherwise there is collision with folder name and pkg name on Pypi
+
+RUN \
     cd lightning && \
     pip install setuptools==59.5.0 && \
-    pip install '.[extra,loggers,strategies]' --no-cache-dir && \
+    PACKAGE_NAME=pytorch pip install '.[extra,loggers,strategies]' --no-cache-dir && \
+    PACKAGE_NAME=lightning pip install '.[extra,loggers,strategies]' --no-cache-dir && \
```

Comment:
(i haven't checked this) since lightning installs `pytorch_lightning` from pypi, could this order be a problem? should PACKAGE_NAME=pytorch be installed after PACKAGE_NAME=lightning?